lang/ar.js

```diff
@@ -2,44 +2,46 @@
 
 export default {
 	"components.alert.close": "إغلاق التنبيه",
-	"components.breadcrumbs.breadcrumb": "فتات الخبز",
+	"components.breadcrumbs.breadcrumb": "شريط التنقل",
 	"components.calendar.notSelected": "لم يتم التحديد.",
 	"components.calendar.selected": "تم التحديد.",
 	"components.calendar.show": "إظهار {month}",
 	"components.dialog.close": "إغلاق",
-	"components.form-element.defaultError": "{label} غير صالح.",
+	"components.form-element.defaultError": "{label} غير صالحة.",
```

Comment:
Was "not good" now is "invalid". Seems fine.

Reply:
Its a bit interesting because this came straight from the LMS